docs/guides/Basics.md

```
@@ -206,43 +238,50 @@ async with ctx.scope(
-    |  ⌊
-    |
-    |  ⎡ @nested [aa01c8e4dad44122bfa79447d964f016](0.00s):
-    |  |  ⎡ •BasicState:
-    |  |  |  ├ identifier: "recorded-nested"
-    |  |  |  ├ value: 11
-    |  |  ⌊
-    |  ⌊
-    ⌊
-
-    07/Mar/2025:13:40:54 +0000 [DEBUG] [logging] [73243ab2691a4753a85a4ddf23643b47] [logging] [488ece9d384e4b6fac258d87ca78d52c] ...exiting context after 0.00s
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)